makiflow/layers/trainable_layers.py

```diff
@@ -77,9 +77,32 @@ def _forward(self, x):
     def _training_forward(self, X):
         return self._forward(X)
 
+
+    @staticmethod
+    def build(params: dict):
+        name = params['name']
+
+        kw = params['shape'][0]
+        kh = params['shape'][1]
+        in_f = params['shape'][2]
+        out_f = params['shape'][3]
+
+        stride = params['stride']
+        padding = params['padding']
+        activation = ActivationConverter.str_to_activation(params['activation'])
+
+        init_type = params['init_type']
+        use_bias = params['use_bias']
+
+        return ConvLayer(
+            kw=kw, kh=kh, in_f=in_f, out_f=out_f,
+            stride=stride, name=name, padding=padding, activation=activation,
+            kernel_initializer=init_type, use_bias=use_bias
+        )
+
     def to_dict(self):
         return {
-            'type': 'ConvLayer',
+            'type': ConvLayer.TYPE,
             'params': {
```

Comment:
Use the constant PARAMS. The same is for all the other layers.
Also replace the 'type' string with the constant.
MakiRestorable.FIELD_PARAMS (change the name of the PARAMS static field)
MakiRestorable.FIELD_TYPE